root-html-file/index.html

```diff
@@ -18,6 +18,8 @@
       }
     </script>
     <link rel="preload" href="https://cdnjs.cloudflare.com/ajax/libs/single-spa/4.3.5/system/single-spa.min.js" as="script" crossorigin="anonymous" />
+    
+    <script src='https://unpkg.com/core-js-bundle@3.1.4/index.js'></script>
```

Comment:
Could you switch this over to the minified version of core-js? It saves 300kb of download.

https://unpkg.com/core-js-bundle@3.1.4/minified.js